test/Dapr.Actors.AspNetCore.Test/ActorHostingTest.cs

```diff
@@ -0,0 +1,84 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+using System.Linq;
+using Dapr.Actors.Runtime;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Dapr.Actors.AspNetCore

```

Comment:
.Test in the namespace like other tests?

Reply:
I didn't do that because it's not a useful practice. Putting tests in a different namespace than the library code just results in more usings. 

If you think it's important I can put it back, for consistency - but I think it's overall not a useful thing.

Reply:
Just a consistency thing....I am fine either ways.